internal-client/src/main/java/org/apache/uniffle/client/request/RssGetShuffleIndexRequest.java

```diff
@@ -47,7 +47,7 @@ public RssGetShuffleIndexRequest(
   @VisibleForTesting
   public RssGetShuffleIndexRequest(
       String appId, int shuffleId, int partitionId, int partitionNumPerRange, int partitionNum) {
-    this(appId, shuffleId, partitionId, partitionNumPerRange, partitionNum, 1, 0);
+    this(appId, shuffleId, partitionId, partitionNumPerRange, partitionNum, 3, 1000);
```

Comment:
Why do you change this?

Reply:
Reverted.

Reply:
We can put them into a new class together.